docs/core/porting/project-structure.md

```diff
@@ -0,0 +1,79 @@
+# Organizing Your Project to Support .NET Framework and .NET Core
+
+This article is to help project owners who want to compile their solution against .NET Framework and .NET Code side-by-side.  It provides several options to organize projects to help developers achieve this goal.
+
+## General Guidance
```

Comment:
Thank you for noticing that. It's not a process. I fixed it with your suggestions